packages/utils/telemetry-utils/src/config.ts

```diff
@@ -4,6 +4,7 @@
  */
 import { ITelemetryBaseLogger, ITelemetryLogger } from "@fluidframework/common-definitions";
 import { Lazy } from "@fluidframework/common-utils";
+import { pkgVersion } from "./packageVersion";
```

Comment:
i've been thinking about something similar, but i don't this this is the correct approach. this package version here will not be consistent across the layers, as this will be loader, not driver or runtime. we also have an issue that any change in version, patch, update, etc will essentially disable all existing configurations.

Reply:
I think this problem needs to be tackled more wholistically in terms about how we are thinking about versions across our layers and data a rest. I hope we can find a workable scheme that is simple, like using integer generations which are easy to define as base lines. 

for instance, in the case of configurations we'd express the config value in a way that expresses for all generations after x, apply this config, maybe like `<generation>.My.Config`. 

Maybe that will be too simple, and we need something more expressive like a semantic version, and sem ver (~,^). But regardless of the solution, this seems like more general problem we need to invest in.